fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -74,6 +74,22 @@ unsigned EligibilityCounter::getCount(int combinedType) const {
 
 } // namespace data_distribution
 
+int getNChooseKLowerBound(int n, int k) {
+	if (k > n)
+		return 0;
+	if (k * 2 > n)
+		k = n - k;
+	if (k == 0)
+		return 1;
+
+	int result = n;
+	for (int i = 2; i <= k; ++i) {
+		result = result * (n - i + 1);
```

Comment:
Are the overflow issues considered here please? If we want to keep the code simple, at least a meaningful error when overflow happens?

Reply:
> Are the overflow issues considered here please? If we want to keep the code simple, at least a meaningful error when overflow happens?
Nice catch! Thanks!